### ISSUE TITLE
Issue after the installation of 'Xamarin.Android.Support.Core.UI'

### ISSUE THREAD

**Issue by TohWei** [alexrainman/CarouselView#82]
/Library/Frameworks/Mono.framework/External/xbuild/Xamarin/Android/Xamarin.Android.Common.targets: Error: java.lang.IllegalArgumentException: already added :  Landroid/support/v4/view/VelocityTrackerCompat; (TestCarouselView2.Droid)

I started with the installation of CarouselView.FormsPlugin. I received a complain about the needs to install this after the 1st build: Xamarin.Android.Support.Core.UI.
However, after the this package (Xamarin.Android.Support.Core.UI), I hit the above error message. 

May I know how to resolve this?

**Comment by kikolopes90** [comment]
Bug: Update v3.0.2
Exception while loading assemblies: System.IO.FileNotFoundException: Could not load assembly 'Xamarin.Android.Support.Core.UI, Version=1.0.0.0, Culture=neutral, PublicKeyToken='. Perhaps it doesn't exist in the Mono for Android profile?

I came back to update v3.0.1 and it works.

**Comment by alexrainman** [comment]
The latest version of the plugin requires Xamarin.Android.Support.v4 version 24.2.1

You can manually update.

**Comment by alexrainman** [comment]
If my work is helping you, please help me back: https://xamarinhq.wufoo.com/forms/nominate-a-xamarin-mvp/

This is what i have done that is community visible:

- Alex has been pretty active sharing his knowledge and answering questions in StackOverflow, GitHub and Xamarin Forums.
- Alex has been open sourcing Xamarin components and plugin with the community, with more of 10 repositories being maintained on GitHub.
- One of Alex's Xamarin.Forms plugin, the CarouselView, is currently being pulled into the Xamarin.Forms core (https://github.com/xamarin/Xamarin.Forms/pull/853). As an stable widget, this control has become the standard for Xamarin.Forms carousel screens with more that 16K downloads in NuGet and more than 100 solved and not open issues.
- Another Alex's plugin, the SegmentedControl (https://github.com/alexrainman/SegmentedControl), is a big candidate to be also pulled into Xamarin.Forms core.
- Plus Xamarin.Forms UI related plugins, Alex shares plugins to set app icon Badge, Google Analytics and KeyChain access.
- Alex shares his knowledge about cutting edge technologies like Couchbase NoSQL and MobileIron integration with Xamarin Apps.

**Comment by ctrexler** [comment]
When I try to update Xamarin.Android.Support.<anything> I get the same error that you're saying I need to update to prevent from occurring. Any ideas?

**Comment by alexrainman** [comment]
The latest version uses latest Xamarin.Android.Support 25, so you can update to latest one.

**Comment by ctrexler** [comment]
Yeah it turns out I just didn't have the A7 sdk installed that would allow me to upgrade to Support 25. I did all that and re-added your assembly and it's working very well, thank you for this! 😄 

One suggestion: Is there any reason you have the default height of the carousel as zero? I thought there was another issue when nothing showed up, but when I changed the background color and height just to see what might happen, I could suddenly see the carousel and the contents I set up for it. The suggestion would be to change it to a default of 200 or something.
